testapp/src/main/res/layout/item_list_manifest_permission.xml

```diff
@@ -24,16 +23,30 @@
         android:layout_centerInParent="true"
         android:orientation="vertical">
 
-        <TextView
-          android:id="@+id/manifestPermissionName"
-          android:textStyle="bold"
-          android:layout_width="wrap_content"
+        <LinearLayout
+          android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:textColor="@android:color/black"
-          android:textSize="@dimen/text_large_16"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent"
-          tools:text="Manifest Permission" />
+          android:orientation="horizontal">
+
+          <TextView
```

Comment:
We also need to display the "Optional" permissions on the first-time launch screen. Optional permissions should be shown with a toggle switch similar to the permissions management screen.

Reply:
Yeah, I am displaying both "required" & "optional" permissions according to the following ux, basically, I am currently using a single adapter to manage both. should I improve here something e.g. mentioning "optional" keyword also? Sorry, previously I haven't asked enough idea for this part...

<img width="308" alt="Screen Shot 2021-03-02 at 19 59 17" src="https://user-images.githubusercontent.com/66930373/109643070-1656d680-7b97-11eb-9f56-8321e151424d.png">